schema_based_on_diagram.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE patients (
+    id INTEGER, 
+    name VARCHAR, 
+    date_of_birth DATE, 
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE medical_histories(
+    id INTEGER, admitted_at TIMESTAMP, 
+    patient_id INTEGER, 
+    status VARCHAR, 
+    FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+CREATE INDEX medical_histories_asc ON medical_histories (patient_id ASC);
+
+CREATE TABLE treatments(
+    id INTEGER, 
```

Comment:
- [ ] While you make a great effort 😃 I see that you forgot to add all "join table" for the many-to-many relastionships is created, even if they are not showed on the diagram. There fore student should create a table between medical_histories and treatments  (name can be different): 💯😊

- Hint: 

```snippet
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );


```

- After adding the relation 😊💯 it will be better if you make sure to add the index for the ids to stick to the requirements 😊🚀:
the indexes should be added to medical_history_id, treatment_id) 👌

Hint: 
```snippet
* `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
* `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`
```